### PR TITLE
Node js pt br

### PR DESCRIPTION
**Pull request by thalesf**
## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

## For resources
free-courses-pt_BR.md

### Description 
Javascript and NodeJS courses in pt-BR.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-pt_BR.md [hunk 1]
```diff
@@ -59,11 +61,22 @@
...
+
+* [Fast & Furious](https://www.youtube.com/watch?v=yRr8Wo4XfYY&list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts (Youtube)
+
+
 ### Kotlin
 
 * [Desenvolvedor Kotlin Iniciante](https://www.udemy.com/desenvolvedor-kotlin-iniciante/) - Gabriel Ferrari, Adriano Sacardo (Udemy)
 
 
+### Node.js
```
Comment (eshellman) [inline free-courses-pt_BR.md]: If you want Node to be a subcategory of javascript, add another # and move it to the javascript section. 

Or... leave Node as its own section and put the heading after Kotlin in the contents list.

File: free-courses-pt_BR.md [hunk 2]
```diff
@@ -59,11 +61,22 @@
 * [Introdução ao Java e Orientação a objetos](https://www.udemy.com/introducao-ao-java-e-orientacao-a-objetos/) - Helder Guimaraes Aragao (Udemy)
 
 
+### Javascript
+
+* [Fast & Furious](https://www.youtube.com/watch?v=yRr8Wo4XfYY&list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts (Youtube)
```
Comment (eshellman) [inline free-courses-pt_BR.md]: If you look at other Youtube links in our course  lists, they point to playlists, for example

https://www.youtube.com/playlist?list=PLXFk6ROPeWoAvLMyJ_PPfu8oF0-N_NgEI

Is there a playlist link for "Fast & Furious" and the Node courses?

File: free-courses-pt_BR.md (hunk 1 above)
Comment (thalesf) [reply free-courses-pt_BR.md]: I put Node in your own category.

File: free-courses-pt_BR.md (hunk 2 above)
Comment (thalesf) [reply free-courses-pt_BR.md]: Yes, there is. Thanks for your feedback!
I updated all links.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**